### PR TITLE
more steps of nextjs tutorial

### PR DESCRIPTION
**Pull request by Joanartdacosta**
- assets, metadata and CSS module of nextjs tutorial steps

### CODE REVIEW

File: public/images/profile.jpg
Comment (doublethink13) [inline public/images/profile.jpg:1]: please do not commit images

use aws bucket

File: pages/components/layout.js [hunk 1]
```diff
@@ -1,5 +1,69 @@
+import Head from "next/head";
+import Image from "next/image";
 import styles from "./layout.module.css";
+import utilStyles from "../../styles/Utils.module.css";
+import Link from "next/link";
 
-export default function Layout({ children }) {
-  return <div className={styles.container}>{children}</div>;
+const name = "Chalet Interior Design";
+export const siteTitle = "Chalet WebApp";
```
Comment (doublethink13) [inline pages/components/layout.js:8]: if this is a constant meant to be reused in other components, please place it in a files called `constants.js` in the root of the project

File: public/images/profile.jpg
Comment (Joanartdacosta) [reply public/images/profile.jpg:1]: I am following the next.js tutorial rules for now. When I finish the tutorial I will adapt according the project

File: pages/components/layout.js (hunk 1 above)
Comment (Joanartdacosta) [reply pages/components/layout.js:8]: for now I am following the next.js tutorial. When I finish, I will adapt according the project

**Pull request merged**